src/Gemstone.Data/Model/TableOperations.cs

```diff
@@ -1280,9 +1749,8 @@ private T LoadRecordWithKeys(DataRow row)
                 {
                     object? value = row.ConvertField(s_fieldNames[property.Name], property.PropertyType);
 
-                    // TODO: Fix encryption
-                    //if (s_encryptDataTargets is not null && value is not null && s_encryptDataTargets.TryGetValue(property, out string keyReference))
-                    //    value = value.ToString().Decrypt(keyReference, CipherStrength.Aes256);
+                    if (s_encryptDataTargets is not null && value is not null && s_encryptDataTargets.TryGetValue(property, out string? keyReference))
+                        value = value.ToString()!.Decrypt(keyReference, CipherStrength.Aes256);
```

Comment:
Uh.. did you fix encryption? What was wrong with it?

Reply:
Encryption wasn't implemented when `Gemstone.Data` was added - but this functionality exists now.